### DISCUSSION TITLE
vector search 接口最大检出的数据量是多少

### DISCUSSION THREAD

**Discussion by Liam-Ji** [milvus-io/milvus#35717] [Q&A and General discussion] [answered]
在v2.4.0版本下
search接口不传limit参数时，检出10条数据
传limit参数，根据文档
![image](https://github.com/user-attachments/assets/a4ad098d-c84f-4075-96b9-bdfde5534d32)
最大支持16384，
请问有什么方式可以一次检出更多数据？

**Comment by yhmo** [comment, marked as answer]
使用iterator分批获取全量数据
https://milvus.io/docs/with-iterators.md#With-Iterators
